player_part.lua

```diff
@@ -64,7 +64,7 @@ end)
 function irc:sendLocal(message)
         for name, _ in pairs(self.joined_players) do
 		minetest.chat_send_player(name, message)
-		irc:logChat(message, name)
 	end
+	irc:logChat(message, name)
```

Comment:
name is `nil` here.  You're just calling `irc:logChat(message)`.  The code for handling a name can be removed.